tests/test_bgt.py

```diff
@@ -121,3 +121,94 @@ def test_feedback_question_year_of_game(
     incorrect_feedback = Feedback(games=games_data, answer=fake_response)
     assert not incorrect_feedback.is_correct
     assert incorrect_feedback.response_text == f"Sorry, the answer was {correct_answer}, not {incorrect_answer}"
+
+
+
+def test_feedback_question_game_by_dev(
+    sample_game, sample_question_game_by_dev, games_data
+):
+
+    correct_answer = extract_attr(
+        game=sample_game,
+        attr=sample_question_game_by_dev.template()["answer_type"]
+    )
+    fake_response = Answer(
+        question=sample_question_game_by_dev,
+        given_answer=correct_answer
+    )
+    correct_feedback = Feedback(
+        games=games_data,
+        answer=fake_response
+    )
+    assert correct_feedback.is_correct
+    assert correct_feedback.response_text == f"Yes, {correct_answer} is right"
+    other_answers = sample_question_game_by_dev.answers
+    other_answers.remove(correct_answer)
+    incorrect_answer = random.choice(tuple(other_answers))

```

Comment:
I don't recommend implementing the following feedback, just considering it for the future:

When it's performant enough to be exhaustive, I think it's better to avoid stochasticity in tests.  In this case, I'd consider writing two tests that use the same set of fixtures. One test shows how correct feedback works and one test shows how incorrect feedback works for each meaningfully different class of incorrect answer (ie do they take separate codepaths?). Maybe there is only one class of incorrect answer, in which case I'd generate that answer as a fixture that can be referred to by name. That answer would then be supplied to both your feedback fixtures and your test.